mailpoet/assets/js/src/segments/dynamic/editor.tsx

```diff
@@ -48,6 +52,12 @@ export function Editor(): JSX.Element {
               icon={chevronLeft}
               href={`#${returnPage}`}
               label={__('Return to previous page', 'mailpoet')}
+              onClick={(event) => {
```

Comment:
Instead of using the `onClick` event to return to the newsletter page, would it make sense to combine this logic with the logic above to define the value of `returnPage`? My first reaction without much thought is that returning to the newsletter page should be another possible value of `returnPage`.

Reply:
Fair point, thanks for looking into it.